### DISCUSSION TITLE
Restriciting Location Query

### DISCUSSION THREAD

**Discussion by trollzkingopaf** [smeijer/leaflet-geosearch#369] [Q&A]
I need help with restricting the location query with OpenStreetMap provider to a specfic country using GeoSearchControl and leafletJS.

**Comment by SoranDK** [comment]
If I understand you right you just add in the parameter to OpenStreetMapProvider:

`provider: new OpenStreetMapProvider({
      params: {
        countrycodes: "gb"
      }
    })`

Above would restrict searches to Great Britain...
